### ISSUE TITLE
Load css script

### ISSUE THREAD

**Issue by pomartel** [MiguelCastillo/load-js#17]
Hi Miguel! Thanks for the very useful library. Works like a charm! I also have this use case where I would like to load css script as well. Is that a scenario you thought about? Would you consider a pull request?

**Comment by MiguelCastillo** [comment]
Hi @pomartel - Im glad you find this library useful! :)

Do you mind helping me understand what you mean by loading "css script"?  Do you have a sample?  Or is there a chance that you mean loading css via a style tag using JavaScript?

**Comment by pomartel** [comment]
Yes, sorry for being vague, I meant loading css via a style tag. Here's what I came up with:

https://github.com/pomartel/load-js/commit/4251646416c0c96a46d7d5a8bcc4736310701d58

And then I can do something like this this (coffeescript) :
```
loadJS([{
    url: '//cdnjs.cloudflare.com/jquery.datetimepicker.full.min.js'
  }, {
    type: 'text/css',
    url: '//cdnjs.cloudflare.com/jquery.datetimepicker.min.css'
  }]).then ->
    $('.datetime').datetimepicker()
```
I have not yet tested it throughly but it seems to work fine.

**Comment by MiguelCastillo** [comment]
Hi @pomartel - I am really sorry I didnt respond.  To me it seems like while it would add better ergonomics as a user, loading CSS the way you are describing would be confusing because simply the name of this module kinda dictates what is meant to do.

But - I can totally see an abstraction on top of this that allows you to load CSS and JS.  Perhaps in the shape of another module `load-resource` or something.  And the API will allow you to load both types.  Any thoughts on that approach?  I already have some other logic to load css assets as well, so it would not be too difficult to put something together.  Any interest in collaborating?

**Comment by pomartel** [comment]
No worries about the delay! I think you're right about the confusion of loading css in a load-js library. I guess my main motive was that quite a few JS libs bundle a separate CSS file that needs to be dynamically loaded too. I've been running my load-js branch with CSS loading in production for the last 2 months and it's solid.

A more generic approach sounds good. Did you have other types of resources in mind beside JS and CSS? Do you want to start a fork of load-js?

**Comment by MiguelCastillo** [comment]
I wasnt thinking of a fork but rather a new module that exposes things like `load.js()` and `load.css`... Perhaps `load.font()` or even web components. Are there assets you think would be useful to have?

**Comment by pomartel** [comment]
Yes, that sounds awesome! I only need css and js for my current web app but this sort of extensibility would be great.

**Comment by tabrez500** [comment]
Hello, is there any update on this :smile: ? Thanks
